LICENSE.md

```diff
@@ -1 +1,295 @@
+- **CartoBCN**: source [data](http://w20.bcn.cat/cartobcn/) and [license](http://w133.bcn.cat/geoportal/descargas/en_gb_cond_us_carto.pdf)
+- **Institut Cartogràfic i Geològic de Catalunya**: source [data](https://github.com/OpenICGC/limits_administratius/tree/master/data) and [license](http://www.icgc.cat/en/The-ICGC/Public-Information/Transparency/Re-use-of-the-information)
+- **NLS National Land Survey of Finland**: source [data](http://www.maanmittauslaitos.fi/en/digituotteet/municipal-division-finland) and [license](http://www.maanmittauslaitos.fi/en/opendata-licence-cc40)
+- **Open Data France**: source [data](https://www.etalab.gouv.fr/wp-content/uploads/2014/05/Open_Licence.pdf) and [license](https://www.data.gouv.fr/en/terms/)
+- **London Datastore - Greater London Authority**: source [data](https://data.london.gov.uk/dataset/statistical-gis-boundary-files-london) and [license](http://www.nationalarchives.gov.uk/doc/open-government-licence/version/2/)
+- **GeoNames**: source [data](http://www.geonames.org/) and [license](http://www.geonames.org/about.html)
+- **Yahoo! GeoPlanet**: source [data](http://developer.yahoo.com/geo/geoplanet/) and [license](http://developer.yahoo.com/geo/geoplanet/data/)
+- **Hong Kong Open Data Portal**: source [data](https://data.gov.hk/en/) and [license](https://data.gov.hk/en/terms-and-conditions)
+- **Helsinki City Real Estate Department**: source [data](http://www.hel2.fi/tietokeskus/data/kartta_aineistot/PKS_Kartta_Rajat_KML2011.zip) and [license](http://www.hri.fi/dataset/paakaupunkiseudun-aluejakokartat)
+- **Ordnance Survey Ireland**: source [data](https://data-osi.opendata.arcgis.com/datasets/administrative-counties-ungeneralised-osi-national-administrative-boundaries-) and [license](https://data-osi.opendata.arcgis.com/datasets/)
+- **International Organization for Standardization**: source [data](http://www.iso.org/) and [license](https://www.iso.org/privacy-and-copyright.html)
+- **International Telecommunications Union**: source [data](https://www.itu.int/online/mm/scripts/gensel8) and [license](https://www.itu.int/en/Pages/copyright.aspx)
+- **City of Kuopio**: source [data](https://www.avoindata.fi/data/fi/dataset/kuopion-kaupunginosat/resource/6ca89290-3743-4832-9ed6-03d8cf9b2d5f) and [license](https://www.avoindata.fi/data/fi/dataset/kuopion-kaupunginosat)
+- **City of Los Angeles Neighborhood Councils (Certified)**: source [data](https://data.lacity.org/A-Well-Run-City/Neighborhood-Councils-Certified-/fu65-dz2f) and [license](https://data.lacity.org/A-Well-Run-City/Neighborhood-Councils-Certified-/fu65-dz2f/about)
+- **Mapshaper**: source [data](https://github.com/mbloch/mapshaper) and [license](http://mozilla.org/MPL/2.0/)
+- **Mesoshapes**: source [data](https://github.com/whosonfirst-data/whosonfirst-data/blob/master/LICENSE.md) and [license](https://github.com/whosonfirst-data/whosonfirst-data/blob/master/LICENSE.md)
```

Comment:
Can you add extra words for sources like Mesoshapes that are themselves compendiums?

Reply:
`This source includes CC-BY compatible data from the following organizations:`

Reply:
fixed in 5827205